src/rhsm/connection.py

```diff
@@ -95,7 +95,7 @@ def drift_check(utc_time_string: str, hours: int = 1) -> bool:
             utc_datetime = dateutil.parser.parse(utc_time_string)
             # This should not have a timezone, but we know it will be utc.
             # We need our timezones to match in order to compare
-            local_datetime = datetime.datetime.utcnow().replace(tzinfo=utc_datetime.tzinfo)
```

Comment:
be careful here: this `local_datetime` will have the tzinfo object not from `datetime` but from `dateutil`, which helps when comparing it against `utc_datetime` (returned by the `dateutil.parser.parse()` above)

I'm afraid the tzinfo object still needs to be replaced

Reply:
Fixed. In theory we could get rid of the whole `dateutil` dependency, but that would be a separate card.